pkg/doozer/prepare/dockerfile.go

```diff
@@ -93,6 +94,10 @@ func NewDockerFile(dockerSpec global.DockerSpec, auroraVersion runtime.AuroraVer
 		env := createEnv(auroraVersion, dockerSpec.PushExtraTags, imageBuildTime)
 
 		dockerFileTemplate := dockerFileTemplateBody
+		if strings.Contains(baseImage.GetCompleteDockerTagName(), "rumple") {
```

Comment:
De gangene vi tidligere har hatt behov for å gjøre feature-switching, så har vi brukt annoteringer i baseimage. Fordelen er at vi ikke binders oss til basenavn, og unngår subtile bugs ved f.eks. nytt baseimage som heter _dumplerumplefumple_ elns

Reply:
Det er en helt grei endring det egentlig - men dette er en midlertidig endring i en versjon av bygging i architect som uansett ikke er relevant for fremtiden, derav least effort.

Reply:
Hvis dette ikke er en endring som skal prodsettes, hvorfor ønsker man den merget til master?

Reply:
Du tenker vi kan spesifisere en spesifikk branch versjon i pipeline eller? Og bare la branchen leve til den ikke trengs mer?